slides/src/content/components.md

```diff
@@ -283,14 +283,14 @@ Notes:
 
 ---
 
-## Template Projection
+## Template Projection (1/3)
```

Comment:
Given the vertical size restriction of the slides, using the title in this way (1/3, 2/3 and 3/3) is not very productive. I tend to think of the title as the first item in the slide. Does it makes sense?